bin/add-idam-clients.sh

```diff
@@ -6,7 +6,7 @@ dir=$(dirname ${0})
 
 ${dir}/utils/idam-create-service.sh "ccd_gateway" "ccd_gateway" "ccd_gateway_secret" "http://localhost:3451/oauth2redirect" "false" "profile openid roles"
 
-${dir}/utils/idam-create-service.sh "xui_webapp" "xui_webapp" "xui_webapp_secrect" "http://localhost:3455/oauth2/callback" "false" "profile openid roles manage-user create-user"
+${dir}/utils/idam-create-service.sh "xuiwebapp" "xuiwebapp" "OOOOOOOOOOOOOOOO" "http://localhost:3455/oauth2/callback" "false" "profile openid roles manage-user create-user"
```

Comment:
callback URL does not match value being added to `bin/env_variables_all.txt`

Reply:
FYI: if this is the correct value to allow XUI docker to work then please update `bin/env_variables_all.txt` and the equivalent files in: aac-manage-case-assignment project. i.e. 
* https://github.com/hmcts/aac-manage-case-assignment/blob/master/aca-docker/bin/add-idam-clients.sh
* https://github.com/hmcts/aac-manage-case-assignment/blob/master/aca-docker/bin/env_variables_all.txt

Reply:
I can confirm that the values here now correspond to the equivalent in hmcts/aac-manage-case-assignment#337